db/psql.py

```diff
@@ -0,0 +1,41 @@
+# import some python ORM for SQl
```

Comment:
what is this comment describing?

Reply:
nothing, it can be deleted

Reply:
fixed https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/64/commits/3dbd4d583fc9b9251c272e8b7250721ed63a56b4